gamemodes/terrortown/gamemode/client/cl_voice.lua

```diff
@@ -499,17 +541,11 @@ VOICE.ScalingFunctions = {
     linear = VOICE.LinearToLinear,
 }
 
-VOICE.GetScalingFunctions = function()
-    local opts = {}
-    for mode in pairs(VOICE.ScalingFunctions) do
-        opts[#opts + 1] = {
-            title = LANG.TryTranslation("label_voice_scaling_mode_" .. mode),
-            value = mode,
-            select = mode == scaling_mode:GetString(),
-        }
-    end
-    return opts
-end
+VOICE.ScalingFunctionChoices = util.ComboBoxChoicesFromKeys(
+    VOICE.ScalingFunctions,
+    "label_voice_scaling_mode_",
+    scaling_mode:GetString()
+)
```

Comment:
Let's quickly move this to the menu code inline, so UI related code is kept away from the business logic.
Also thx for the nice util function!

Reply:
makes sense, I moved the convar objects into a globally accessible VOICE.cv table to achieve this